project.clj

```diff
@@ -7,20 +7,19 @@
             :url          "http://www.eclipse.org/legal/epl-v10.html"
             :distribution :repo
             :comments     "same as Clojure"}
-  :plugins [[lein-parent "0.3.7"]]
-  :clean-targets [:target-path :compile-path "out"]
-  :parent-project {:coords  [io.jesi/parent "3.3.0"]
+  :plugins [[lein-parent "0.3.8"]]
+  :clean-targets ^{:protect false} [".shadow-cljs" "out" :target-path]
+  :parent-project {:coords  [io.jesi/parent "3.9.0"]
                    :inherit [:plugins :managed-dependencies :deploy-repositories :dependencies :profiles :test-refresh :aliases :codox]}
   :managed-dependencies [[com.google.guava/guava "23.0"]
-                         [io.jesi/backpack]]
-  :dependencies [[io.jesi/backpack "4.2.1"]                 ;TODO use managed version (at least 5.0)
+                         [io.jesi/backpack "5.1.2-SNAPSHOT"]] ;FIXME use managed 5.1.2 version
+  :dependencies [[io.jesi/backpack]
                  ;CLJ
                  [org.clojure/clojure :scope "provided"]
                  [leiningen "2.9.1" :exclusions [org.slf4j/slf4j-nop]]
                  [org.clojure/tools.namespace "0.3.1"]
                  ;CLJS
-                 [org.clojure/clojurescript :scope "provided"]
-                 [thheller/shadow-cljs :scope "provided"]
+                 [thheller/shadow-cljs]
```

Comment:
Should this be a dependency? Or a dev dependency?

Reply:
Done.
Yup, dev dependency and `clojurescript` as provided